src/components/common/button/index.js

```diff
@@ -0,0 +1,29 @@
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+
+class Button extends PureComponent {
+	constructor(props) {
+		super(props);
+		this.onClick = this.onClick.bind(this);
+	}
+
+	onClick(e) {
```

Comment:
This method does not do anything, `this.props.onClick` can be passed to `button` directly